joomla/joomla-cms
# [4.0] Template Atum - Change Main Color Names and Variables

### Summary of Changes
Renamed everything what has in common with the bg-dark color to Primary colour and the special color to secondary color. Moved the secondary color into the first column ob the backend settings.

### Testing Instructions
Apply the patch
npm run build:css
The color names in the setting should be now primary and secondary color and everything should work just like before the patch.

<img width="432" alt="grafik" src="https://user-images.githubusercontent.com/828371/81208527-7ad4f380-8fcf-11ea-9d9e-14bdc516c966.png">

### Reason of the change

Every modern framework uses the terms _Primary Color_ and _Secondary color_ and for increased usability I suggest to change the naming in Joomla. 

Also it does not fix a color to have a specific brightness. Since the dark sidebar is gone and we don't need anymore to calculate all the shades of the sidebar for a11y another step would be also to remove the hue slider and replace it by a colorfield where the user is free to chose the color.
